### PR TITLE
Examples/switch space

### PR DESCRIPTION
**Pull request by davidhayes9**
Adding an example to get switches with /mnt/flash /var/log or /var/core that are over 90% utilised

### CODE REVIEW

File: examples/Connector/get_switches_full_dir.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2020 Arista Networks, Inc.
```
Comment (cianmcgrath) [inline examples/Connector/get_switches_full_dir.py]: 2024*

**Review by cianmcgrath — CHANGES_REQUESTED** [review]
The change fails the linting checks. They can be run locally with `make lint`.
Issues: 
```
make lint
flake8 .
./examples/Connector/get_switches_full_dir.py:13:1: E302 expected 2 blank lines, found 1
./examples/Connector/get_switches_full_dir.py:29:44: W291 trailing whitespace
./examples/Connector/get_switches_full_dir.py:30:65: W291 trailing whitespace
./examples/Connector/get_switches_full_dir.py:79:5: F841 local variable 'query' is assigned to but never used
./examples/Connector/get_switches_full_dir.py:85:1: W293 blank line contains whitespace
./examples/Connector/get_switches_full_dir.py:87:22: E225 missing whitespace around operator
./examples/Connector/get_switches_full_dir.py:89:9: E265 block comment should start with '# '
make: *** [lint] Error 1
```
Please address these so this change does not impact the pipelines

**Comment by cianmcgrath** [comment]
If you don't have the tooling for the linting installed you can run `make dev-setup` to install them

**Comment by davidhayes9** [comment]
thanks Cian for reviewing, I have made the changes to resolve the flake8 errors and corrected the date.

**Pull request merged**